srcpkgs/disputils/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'disputils'
+pkgname=disputils
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="xcb-util-devel"
+short_desc="Utilities for getting info about displays"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="ISC"
+homepage="https://github.com/tudurom/disputils"
+distfiles="https://github.com/tudurom/disputils/archive/v${version}.tar.gz"
+checksum=67a07dce4505f15f9b6c770dcd47109a344c5c0f13ad1916785e3894920bc799
+LDFLAGS="-Wl,--no-as-needed"
+
+post_extract() {
+	sed -i -e '/^CC /d' \
+	config.mk
```

Comment:
don't think it needs to be in a new line

Reply:
done